examples/tutorial_dbt_dagster_v2/tutorial_dbt_dagster_v2/assets/dbt_seed_run_test.py

```diff
@@ -0,0 +1,20 @@
+from dagster_dbt.asset_decorators import dbt_multi_asset
+from dagster_dbt.cli.resources_v2 import DbtClientV2, DbtManifest
+
+from tutorial_dbt_dagster_v2.assets import raw_manifest
+
+manifest = DbtManifest(raw_manifest=raw_manifest)
+
+
+## Case 1(c): run dbt seed, run, and test as separate commands.
```

Comment:
I think we should take out these numbers and letters, because they'll be difficult to maintain